src/app/components/login/verification/verification.component.scss

```diff
@@ -0,0 +1,65 @@
+@import url("./../login.component.scss");
```

Comment:
Do not import styles from other components. If you have shared styles then move them to the scss file above both components in hierarchy (perhaps to `scss` folder in root folder).